### PR TITLE
Include additional links in the header

### PR DESCRIPTION
**Pull request by javidhsueh**
- Able to include the additional links in the header. 
 - Update API documentation.
 - Bump ocular-gatsby version of the website.

### CODE REVIEW

File: docs/gatsby/api-reference/ocular-config.md [hunk 1]
```diff
@@ -41,6 +41,19 @@ These are bullets on the home page, typically the main selling points of your fr
...
+## ADDITIONAL_LINKS
+
+You can add additional links in the header section. These links will be added after the default menu.
+Each link entry has the following fields:
+
+| Field     | Type     | Name  |
+| ---       | ---      | ---   |
+| `label`    | `String` | Required. The displayed text in the header link. |
+| `href`    | `String` | Optional. The hyperlink you want to redirect to. |
+| `to`   | `String` | Optional. The path to a gatsby page |
```
Comment (ibgreen) [inline docs/gatsby/api-reference/ocular-config.md]: Is it either `href` or `to`? Maybe clarify?

File: docs/gatsby/api-reference/ocular-config.md [hunk 2]
```diff
@@ -49,5 +62,5 @@ Each EXAMPLE entry has the following fields
 | ---            | ---      | ---   |
 | `componentUrl` | `String` | The filename of a React (`.jsx`) file |
 | `title`        | `String` | The title of example. |
-| `image`        | `String` | The icon for the bullet. Typically `images/<example-screenshot>.png`. Images are resolved in the `static` folder. |
+| `image`        | `String` | The icon for the bullet. Typically `images/<example-screenshot>.png`. Images are resolved in the `static` folder. (Right now only support png files) |
```
Comment (ibgreen) [inline docs/gatsby/api-reference/ocular-config.md]: This was a recent regression. I opened an ocular-gatsby issue on it. One possibly related change is that I bumped `sharp` from 0.22 to 0.22.1 to support Node 12. Maybe check if more recent fix is now available.

File: modules/gatsby/src/components/layout/header.jsx [hunk 3]
```diff
@@ -127,7 +120,8 @@ export default class Header extends Component {
                 )
               }
             ]
-          : [])
+          : []),
+        ...(props.config && props.config.ADDITIONAL_LINKS)
```
Comment (ibgreen) [inline modules/gatsby/src/components/layout/header.jsx]: Is config sometimes not available?

Comment (javidhsueh) [reply modules/gatsby/src/components/layout/header.jsx]: this is just a sanity check.

File: docs/gatsby/api-reference/ocular-config.md (hunk 1 above)
Comment (javidhsueh) [reply docs/gatsby/api-reference/ocular-config.md]: will do

**Pull request merged**